yoda-toolfile.spec

```diff
@@ -15,7 +15,9 @@ cat << \EOF_TOOLFILE >%i/etc/scram.d/yoda.xml
     <environment name="LIBDIR" default="$YODA_BASE/lib"/>
     <environment name="INCLUDE" default="$YODA_BASE/include"/>
   </client>
-  <use name="boost"/>
+  <use name="python"/>
+  <use name="cython"/>
+  <use name="root_cxxdefaults"/>
```

Comment:
OK, I have now a local version where I reverted the mis-happening changes above and I think the root dependency is behaving as it should (the ldd was showing that it finds root librairies from the CMSSW I was using to test the builded package). So I guess I could proceed with remaking the PR or do we have something else missing ?